### PR TITLE
Message broker

### PR DESCRIPTION
**Pull request by why-not-try-calmer**
This is a simple message broker for handling all messaging things. Let me know if this 'seed' is OK for you, so that I know if / where to water it :)

### CODE REVIEW

File: defrag/modules/helpers/broker.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+from pottery import RedisDeque
+from typing import Any, Coroutine, Dict, Generator, List, Optional
+import asyncio
+
+
+@dataclass
+class Message:
+    timestamp: float
+    sender: str
+    addressee: str
```
Comment (KaratekHD) [inline defrag/modules/helpers/broker.py:23]: Why do we have an addressee? My thought was we would implement some sort of "top to bottom" communication, where we send a message to a lot of people. This looks, combined with the tests, more like a messaging service, which I feel we have no need (and the required skills for making it secure and private) for.

Comment (why-not-try-calmer) [reply defrag/modules/helpers/broker.py:23]: Don't worry, this is a "top to bottom" communication service. We simply need to know which entity sent the message and which entity it is meant for, that is all. "sender" and "addressee" are just simple words for my simple mind to remember easily.

Example:
- sender: "telegram:bot,user:jens"
- addressee: "telegram:subscribed_bots,matrix:subscribed_bots"
- text: "Today there will be an outage due to a meteor impact in the Nuremberg region. Expect 1-2 hours with oS infra down, starting at 12 pm."

Comment (KaratekHD) [reply defrag/modules/helpers/broker.py:23]: Love this message :joy:

**Comment by why-not-try-calmer** [comment]
I squashed all my commits to make it cleaner. I have redesigned and simplified, an finialized the documentation. Happy vacation!

**Comment by why-not-try-calmer** [comment]
How does this look for you @KaratekHD ?

**Review by KaratekHD — CHANGES_REQUESTED** [review]
Looks good, but please add __pycache__ to .gitignore, as it does not need to be in the repo.

**Pull request merged**